aiida/plugins/entry_point.py

```diff
@@ -69,7 +68,13 @@ class EntryPointFormat(enum.Enum):
 }
 
 
-def validate_registered_entry_points():  # pylint: disable=invalid-name
+def parse_entry_point(group: str, spec: str) -> EntryPoint:
```

Comment:
should that perhaps be called `create_entry_point`?
how is this method used?

Calling it `parse` may be a bit confusing together with the `parse_entry_point_string` methods

Reply:
well it was just a replacement of `EntryPoint.parse` in ` aiida/manage/database/integrity/plugins.py`, that is the only place it is used. I thought it would be better to centralise the logic into this module.